README.md

```diff
@@ -1,11 +1,85 @@
+    - `Open diff view`: Open diff view for the current file
+    - `Stage current file`
+    - `Unstage current file`
+- Commit
+    - `Commit all changes`: Only commits all changes without pushing
+    - `Commit all changes with specific message`: Same as above, but with a custom message
+    - `Commit staged`: Commits only files that have been staged
+    - `Commit staged with specific message`: Same as above, but with a custom message
+- Backup
+    - `Create Backup`: Commits all changes. If "Push on backup" setting is enabled, will also push the commit.
+    - `Create Backup with specific message`: Same as above, but with a custom message
+    - `Create backup and close`: Same as `Create Backup`, but if running on desktop, will close the Obsidian window. Will not exit Obsidian app on mobile.
+- Remote 
+    - `Push`
+    - `Pull`
+    - `Edit remotes`
+    - `Remove remote`
+    - `Clone an existing remote repo`: Opens dialog that will prompt for URL and authentication to clone a remote repo
+    - `Open file on GitHub`: Open the file view of the current file on GitHub in a browser window. Note: only works on desktop
+    - `Open file history on GitHub`: Open the file history of the current file on GitHub in a browser window. Note: only works on desktop
+- Local
+    - `Initialize a new repo`
+    - `Create new branch`
+    - `Delete branch`
+    - `CAUTION: Delete repository`
+- Source Control View
+    - `Open source control view`: Opens side pane displaying [Source control view](#sidebar-view)
+    - `Edit .gitignore`
+    - `Add file to .gitignore`: Add current file to .gitignore
+
+## Mobile
+
+### Installation
+
+#### Existing Repo
+
+Follow these instructions for setting up an Obsidian Vault on a mobile device that is already backed up in a remote git repository. 
+
+The instructions assume you are using GitHub, but can be extrapolated to other providers.
+
+1. Make sure any outstanding changes on all devices are pushed and reconciled with the remote repo.
+2. Install Obsidian for Android or iOS.
+3. Create a new vault (or point Obsidian to an empty directory). Do NOT select `Store in iCloud` if you are on iOS.
+4. (Recommended for security) [Create a personal access token on GitHub](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). Minimal permissions required are "Read access to metadata" and "Read and Write access to code and commit statuses" for the repo you are going to use.
+5. In Obsidian settings, enable community plugins. Browse plugins to install Obsidian Git.
+6. Enable Obsidian Git (on the same screen)
+7. Go to Options for the Obsidian Git plugin (bottom of main settings page, under Community Plugins section)
+8. Under the "Advanced" section, fill in the username on your git server and your password/personal access token. NOTE: If using a personal access token (recommended), use `__token__` as your username, not your actual GitHub username.
```

Comment:
Where did you get that `__token__` username from? I'm not aware of that "feature".

Reply:
I've seen it in [this GitHub blog](https://gist.github.com/br3ndonland/f9c753eb27381f97336aa21b8d932be6?permalink_comment_id=3668230#build-and-publish-in-one-step).

When I tried to use my actual GItHub username, it didn't work, but the PAT worked when I used `__token__` as the username.

Reply:
In your linked gist, they use `__token__` as username for `PyPI` and not GitHub. I can't find any other resource regarding GitHub. I have no issues using my username `Vinzent03`

Reply:
I tried again, and it worked with my username this time. It's interesting that both work. I can update it.